scripts/cli/runInDocker.sh

```diff
@@ -59,6 +56,7 @@ run_in_environment_docker() {
 
 run_in_deployment_docker() {
   run_in_docker \
+    -e IN_DEPLOYMENT_CONTAINER=true \
```

Comment:
If we set an environment variable for all CLI containers (in run_in_docker), the startContainer.sh script could work in the builder container as well.

Reply:
Good point, i'll update this for all containers